src/NormalizingFlows.jl

```diff
@@ -57,46 +55,25 @@ function train_flow(
     # otherwise the compilation time for destructure will be too long
     θ_flat, re = Optimisers.destructure(flow)
 
+    loss(θ, rng, args...) = -vo(rng, re(θ), args...)
+
     # Normalizing flow training loop 
-    θ_flat_trained, opt_stats, st = optimize(
-        rng,
+    θ_flat_trained, opt_stats, st, time_elapsed = optimize(
         ADbackend,
-        vo,
+        loss,
         θ_flat,
         re,
-        args...;
+        (rng, args...)...;
```

Comment:
Isn't this just
```suggestion
        rng,
        args...;
```
?

Reply:
ah yes. Don't know what I was thinking lol